### ISSUE TITLE
Provide a screen reader accessible template editor alternative

### ISSUE THREAD

**Issue by MarcoZehe** [podlove/podlove-publisher#716]
The current template editor is not accessible to screen readers. The fact that the text area is only used for event capturing, but all else is somehow done via coloring lines in CSS and other funky visual stuff. This means that blind podcasters using screen readers on Windows or Mac cannot use this editor.

The probably easiest way is to provide an alternative editor that can be invoked via the Options to that particular settings sheet. See the Widget configuration in the Themes section of the regular WordPressAdmin as an example. What should then be presented should be a simple text area that contains the text of the template, indentations and all. Screen readers usually have their own means of dealing with text, so we don't need line numbers or other visual bells and whistles for properly editing text. Just please give us the option to switch to a simple text area for editing templates. Thanks!

**Comment by eteubert** [comment]
It looks like [Ace](http://ace.c9.io/), the underlying editor toolkit, is just not accessible. So I guess we have to offer a switch to a simple textarea like you are suggesting.

Is the rest (adding / selecting / deleting  templates) accessible?

Can you recommend a screenreader for me as a developer to spot accessibility issues?

**Comment by MarcoZehe** [comment]
Yes, the rest is accessible. If you’re on a Mac, you can use VoiceOver, built-in, with Safari or Chrome, to test stuff. Cmd+F5 will toggle. On Windows, with Firefox preferably, but IE or Chrome will do in a more limited way, too, you can use [NVDA](http://www.nvda-project.org <http://www.nvda-project.org/>), but make sure you take a look at keyboard and the Windows screen reader browsing paradigm first. There are introductory videos all over Youtube that should help get you started. Same is true for VoiceOver on OS X.
